api/resolvers/user.js

```diff
@@ -57,6 +57,7 @@ export async function topUsers (parent, { cursor, when, by, from, to, limit = LI
 
   let column
   switch (by) {
+    case 'spending':
```

Comment:
This is the fix for the top spenders API call. I thought this was easiest rather than refactoring all of the URLs which use `spending` instead of` spent`.